data/json/items/tool/landscaping.json

```diff
@@ -69,6 +69,42 @@
     "color": "brown",
     "qualities": [ [ "DIG", 2 ], [ "COOK", 1 ], [ "BUTCHER", -96 ] ]
   },
+  {
+    "id": "rake",
+    "type": "TOOL",
+    "name": { "str": "metal rake" },
+    "description": "A sturdy metal rake, a must-have during autumn.",
+    "weight": "1500 g",
+    "volume": "3500 ml",
+    "longest_side": "150 cm",
+    "price": 1000,
+    "price_postapoc": 150,
+    "to_hit": 2,
```

Comment:
base accuracy: -2
grip: solid +1
length: long +2
surface: any: +0
balance: neutral (+0) or uneven (-1)

should be +1 or +0, probably +0.

damage should probably be:
"bashing": 12,
"cutting": 3,

rakes are clumsy improvised quarterstaffs with a little extra ouch going for them, they're not serious cutting weapons.